comixed-tasks/src/main/java/org/comixedproject/task/model/MoveComicWorkerTask.java

```diff
@@ -120,26 +120,30 @@ String getRelativeComicFilename() {
       return FilenameUtils.getBaseName(this.comic.getFilename());
     }
 
-    log.debug("Generating relative filename based on renaming rule: {}", this.renamingRule);
+    log.debug("Scrubbing renaming rule: {}", this.renamingRule);
+    final String rule = this.scrub(this.renamingRule, "[:\\\\*?|<>]");
```

Comment:
Because these are characters that are globally forbidden and because this list of characters is used in several places, wouldn't it be better to define it as a constant?
We can use this.scrub(this.renamingRule) too, no ?

Reply:
Sure, that works! Fixed it. I also added scrubbing the rules as they're entered on the browser to avoid submitted invalid characters as well.